CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
+## 2019-??-?? - 1.0.3
+  - Added items to satisyf puppet-lin and pdk (Thanks alexjfisher)
```

Comment:
Was this really me?

Reply:
Shoot, you are right. It was silug.

Reply:
and I think I've found another bug.  Just debugging it now.

Reply:
Quite possible. There was a slew of pull requests and at least one broke a few things. We got a correction back in for that, but it's possible other scenarios that the reporter had not reviewed were failing.

Reply:
```
undefined method `key?' for nil:NilClass
/opt/puppetlabs/puppet/cache/lib/puppet/util/xml_file.rb:136:in `block in set_tag'
/opt/puppetlabs/puppet/lib/ruby/2.5.0/rexml/xpath.rb:68:in `each'
/opt/puppetlabs/puppet/lib/ruby/2.5.0/rexml/xpath.rb:68:in `each'
/opt/puppetlabs/puppet/cache/lib/puppet/util/xml_file.rb:112:in `set_tag'
/opt/puppetlabs/puppet/cache/lib/puppet/provider/xml_fragment/xml_fragment.rb:154:in `create'
```

I don't think this is necessarily a regression in the pdk PR though. In fact, I'm pretty sure it's not.  I'll open a PR with a fix.

Reply:
https://github.com/Areson/xml_fragment/pull/12

Reply:
That one isn't, no. One of the previous PRs modified some class variable names and it wasn't propagated throughout the code properly and broke some lookups.

Reply:
yes, I saw that.  I've only hit these issues today as I'm trying to use `purge` for the first time.